public/src/Frame.js

```diff
@@ -0,0 +1,24 @@
+function Frame(){
```

Comment:
Good use of Javascript's constructor function to have separate objects for Frame and Scorecard. Means you are complying with SRP and code is nice and readable.